backend/src/main/kotlin/rocks/didit/sefilm/web/controllers/ShowingController.kt

```diff
@@ -192,31 +193,41 @@ class ShowingController(private val repo: ShowingRepository,
   }
 
   @PostMapping(PATH_WITH_ID + "/attend", produces = arrayOf(MediaType.APPLICATION_JSON_UTF8_VALUE))
-  fun attend(@PathVariable id: UUID): Set<UserID> {
+  fun attend(@PathVariable id: UUID, @RequestBody body: AttendInfoDTO): List<Participant> {
```

Comment:
Suddenly `List<Participant>` instead of a `Set`. Doesn't really matter to me, just be consistent in your usage.